app/src/ui/dispatcher/dispatcher.ts

```diff
@@ -1168,6 +1179,7 @@ export class Dispatcher {
         originalBranchTip
       )
     }
+    return
```

Comment:
I guess this can be removed, no?
```suggestion
```

Reply:
:D It would have been mad at that because this made all paths return something -> But put the return on the complete and switch and it is happy.